packages/dashboard/src/components/sidePanel/shared/styles.css

```diff
@@ -27,3 +27,17 @@
   height: 1.57rem;
   border-radius: 0.57rem;
 }
+
+/* hide default color input */
+.color-picker-container > input {
+  opacity: 0;
```

Comment:
If this is intended to be noninteractive we might want to guarantee that with other things like `pointer-events: none`

Reply:
the input will still be interactive. this was for hiding the default native color picker button as below:
<img width="187" alt="image" src="https://user-images.githubusercontent.com/11740421/215883891-c5b9d7ab-b075-4b9f-b98e-a849c974a551.png">